Cargo.toml

```diff
@@ -30,6 +30,7 @@ once_cell = "1.5.2"
 derive_more = { version = "0.99.14", features = ["display"] }
 anyhow = "1.0.40"
 if_chain = "1.0.1"
+url = "2.2.2"
```

Comment:
I just noticed that deno_ast with `module_specifier` feature enabled has the exported type called `ModuleSpecifier`, which is effectively `url::Url`.
https://github.com/denoland/deno_ast/blob/257bf920fa8011a88bd06d3af858179a139d1ab0/src/media_type.rs#L9-L10

So instead of importing `url` crate here in deno_lint, can we use `deno_ast::media_type::ModuleSpecifier`?

Reply:
I tested it but it don't resolves, and I guess we should use ```use deno_core::ModuleSpecifier;``` but also we don't have deno_core here

Reply:
Oops, how about `deno_ast::ModuleSpecifier` (with `module_specifier` feature enabled)? From my quick test on my local it should work.

Reply:
aa actually it is also unresolvable

Reply:
<img width="804" alt="Screen Shot 2022-02-20 at 13 11 41" src="https://user-images.githubusercontent.com/72156162/154836920-a3fa7af4-9a0a-44b6-ba9e-26e6743f351f.png">

Reply:
Is it? In my environment, adding `module_specifier` feature for `deno_ast` in `Cargo.toml` has resolved that issue. Could you please confirm that `module_specifier` feature is enabled?

Reply:
aa right, yeah I added and it is now available